android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -203,10 +203,7 @@ public void voice_connect_android(String accessToken, ReadableMap twimlParams, P
       .build();
     CallRecord callRecord = new CallRecord(
       uuid,
-      Voice.connect(
-        getReactApplicationContext(),
-        connectOptions,
-        new CallListenerProxy(uuid, reactContext))
+      getVoiceServiceApi().connect(connectOptions, new CallListenerProxy(uuid, reactContext))
```

Comment:
Should we give the call listener proxy the service context instead of the react one?

Reply:
ohh good find, yup